gaeguli/pipeline.c

```diff
@@ -828,6 +833,24 @@ gaeguli_pipeline_update_vsrc_caps (GaeguliPipeline * self)
   }
 }
 
+static void
+gaeguli_pipeline_emit_recording_started (GaeguliPipeline * self,
+    GaeguliRecordTarget * rec_target)
+{
+  g_mutex_lock (&self->lock);
+  ++self->num_active_rec_targets;
+  g_mutex_unlock (&self->lock);
+}
+
+static void
+gaeguli_pipeline_emit_recording_stopped (GaeguliPipeline * self,
+    GaeguliRecordTarget * target)
+{
+  g_mutex_lock (&self->lock);
+  --self->num_active_rec_targets;
+  g_mutex_unlock (&self->lock);
+}
+
```

Comment:
What's the point of these signal handlers? They don't actually emit anything and `num_active_rec_targets` doesn't seem to be used anywhere either.